crates/ra_syntax/src/ast.rs

```diff
@@ -17,8 +17,8 @@ pub use self::{
     generated::*,
     traits::*,
     tokens::*,
-    extensions::{PathSegmentKind, StructKind, FieldKind, SelfParamKind},
```

Comment:
looks like you removed this use, probably while rebasing, that's why it's currently not building